lib/child_process.js

```diff
@@ -195,6 +195,8 @@ exports.execFile = function(file /*, args, options, callback*/) {
 
   var ex = null;
 
+  var cmd = file;
+
```

Comment:
Also /cc @nodejs/v8 and @targos.

I don't think that this is the only place that is affected by our update to V8 5.6, I expect more deopts like this.

This better should be fixed on the V8 side instead of rewriting all the places in Node.js where a named function parameter is used in a child scope.